SPECS/autofs/autofs.spec

```diff
@@ -2,39 +2,38 @@
 Summary:        A kernel-based automounter for Linux
 Name:           autofs
 Version:        5.1.8
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        GPLv2+
-URL:            http://www.kernel.org/pub/linux/daemons/autofs
-Group:          System Environment/Daemons
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:        http://www.kernel.org/pub/linux/daemons/%{name}/v5/%{name}-%{version}.tar.xz
-
-BuildRequires:  systemd
-BuildRequires:  rpcsvc-proto-devel
+Group:          System Environment/Daemons
+URL:            https://git.kernel.org/pub/scm/linux/storage/autofs/autofs.git/
+Source0:        https://www.kernel.org/pub/linux/daemons/%{name}/v5/%{name}-%{version}.tar.xz
 BuildRequires:  libtirpc-devel
-Requires:       systemd
+BuildRequires:  rpcsvc-proto-devel
+BuildRequires:  systemd-devel
 Requires:       libtirpc
+Requires:       systemd
+
 %description
 Automounting is the process of automatically mounting and unmounting of file systems by a daemon. Autofs includes both a user-space daemon and code in the kernel that assists the daemon.
 
 %prep
-%setup -q
+%autosetup
 
 %build
-./configure --prefix=/usr           \
-            --mandir=/usr/share/man \
-           --with-libtirpc
-make %{?_smp_mflags}
+%configure \
+  --with-libtirpc
+%make_build
 
 %install
 mkdir -p -m755 %{buildroot}/lib/systemd/system
-mkdir -p -m755 %{buildroot}/etc/auto.master.d
+mkdir -p -m755 %{buildroot}%{_sysconfdir}/auto.master.d
 make install mandir=%{_mandir} INSTALLROOT=%{buildroot}
 make install_samples INSTALLROOT=%{buildroot}
-make -C redhat
+%make_build -C redhat
```

Comment:
You wouldn't know if this `-C redhat` part is necessary in Mariner, would you?:)

Reply:
It is- that's the folder name in the source